### DISCUSSION TITLE
Failing to add ios - 'CocoaPods could not find compatible versions for pod "Capacitor"'

### DISCUSSION THREAD

**Discussion by stevecassidy** [ionic-team/capacitor#4413] [Q&A] [answered]
I'm just adding ios to my project which is already working ok on Android.  I did:
```
$ sudo gem install cocoapods
$ yarn add @capacitor/ios@next
$ npx cap add ios
```
The output from this last command is:

```
ℹ Installing iOS dependencies – Skipping: already installed
✔ Adding native xcode project in: /Users/Steve/projects/a-wptas/ios in 71.67ms
✔ add in 73.96ms
✔ Copying web assets from build to ios/App/public in 175.19ms
✔ Copying native bridge in 5.52ms
✔ Copying capacitor.config.json in 6.97ms
✔ copy in 250.38ms
✔ Updating iOS plugins in 60.14ms
  Found 0 Capacitor plugins for ios:
✖ Updating iOS native dependencies with "pod install" (may take several minutes): 
✖ update ios: 
[error] Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "Capacitor":
  In Podfile:
    Capacitor (from `../../node_modules/@capacitor/ios`)

Specs satisfying the `Capacitor (from `../../node_modules/@capacitor/ios`)` dependency were found, but they required a higher minimum deployment target.
Ignoring ffi-1.13.1 because its extensions are not built. Try: gem pristine ffi --version 1.13.1
```

I have previously built an ios project on this machine (different project).   

Any hints on where to look for a solution to this would be appreciated. 

Thanks,
Steve

**Comment by stevecassidy** [comment, marked as answer]
The issue proved to be the version of @capacitor/ios that had installed, since I added this a few weeks after setting up the other parts of the project the default version I got was @3.0.0-rc0 (or something like that) where the rest of the codebase was @2.4.6.   Solved by removing the package and reinstalling @capacitor/ios@2.  

Steve

**Reply by jcesarmobile** [reply]
yeah, if you install Capacitor 3 you have to bump the deployment target to iOS 12 
https://capacitorjs.com/docs/v3/updating/3-0#set-ios-deployment-target-to-120
